src/CRM/CivixBundle/Resources/views/Code/upgrader-base.php.php

```diff
@@ -100,7 +106,7 @@ public function executeCustomDataFile($relativePath) {
    *
    * @return bool
    */
-  protected static function executeCustomDataFileByAbsPath($xml_file) {
+  protected function executeCustomDataFileByAbsPath($xml_file) {
```

Comment:
Oh, yeah, the call to `$this->executeCustomDataFileByAbsPath()` doesn't match the official signature. So this fixes the signature.

I grepped `universe` to see if the signature change might be a problem -- fortunately, almost nobody else calls the function directly; and the ones who do are using the `$this->` formulation.

So looks good. :+1: